index.html

```diff
@@ -107,16 +107,26 @@ <h1>EmulatorJS Demo</h1>
 
         <script>
             var enableDebug = false;
+            var enableThreads = false;
```

Comment:
Could we use `let` here

Reply:
Both enableDebug and enableThreads have now been changed to let